terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java

```diff
@@ -33,6 +33,9 @@
 
     @Value("${app.redirect.allowed.externalUrl}")
     String redirectionAllowedExternalUrl;
+    
+    @Value("${app.redirect.pageTitle.404Error}")
+    String pageTitle404Error;

```

Comment:
@ikeyat 
This is related this PR d9b7c75f59c2b97eaae68b7fc5d98dff9427d67c changes of application-env.properties.
I changed the error page to be expected, because Websphere wrap Unexpected Exceptions in ServletException unlike other servers.